src/components/Forside/Forside.tsx

```diff
@@ -0,0 +1,135 @@
+import { Sidetittel, Systemtittel, Undertittel, Ingress } from 'nav-frontend-typografi';
+import Tekstområde, { BoldRule } from 'nav-frontend-tekstomrade';
+import { BekreftCheckboksPanel } from 'nav-frontend-skjema';
+import Lenke from 'nav-frontend-lenker';
+import { Hovedknapp } from 'nav-frontend-knapper';
+import VeilederSnakkeboble from '../../assets/VeilederSnakkeboble';
+import Panel from 'nav-frontend-paneler';
+
+const Forside: React.FC = () => {
+    const LOREM =
+        'Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe sed consectetur maiores veniam voluptas obcaecati, cumque hic laboriosamcorporis vel temporibus optio blanditiis. Dolorum autem exercitationem nulla, sed porro eum!x';
+
+    const [bekreftet, settBekreftet] = useState<boolean>(false);
+
+    const handleOnChange = () => {
+        settBekreftet(!bekreftet);
+    };
+
+    return (
+        <div className={'forside'}>
+            <div className={'forside__innhold'}>
+                <Panel className={'forside__panel'}>
+                    <div className={'veileder'}>
+                        <VeilederSnakkeboble />
+                    </div>
+                    <Sidetittel>Søknad om barnetrygd</Sidetittel>
+
+                    <div className={'ingress'}>
+                        <Ingress className={'ingress__tekstområde'}>{LOREM}</Ingress>
+                        <Lenke href={'https://www.nav.no'}>Mer om barnetrygd</Lenke>
+                    </div>
+
+                    <div className={'opplysninger'}>
+                        <Systemtittel className={'opplysninger__systemtittel'}>
+                            Det er viktig at du gir oss riktige opplysninger
+                        </Systemtittel>
+                        <Tekstområde className={'opplysninger__tekstområde'}>
+                            {`For at vi skal kunne behandle søknaden din, må du gi oss riktige opplysninger. 
+                            
+                            Hvis du får overgangsstønad, må du melde fra når det skjer viktige endringer i livet ditt, for eksempel bo-og familiesituasjonen eller arbeid og utdanning. Det samme gjelder dersom inntekten din endrer seg.`}
+                        </Tekstområde>
+                    </div>
+
+                    <div className={'dokumentasjon'}>
+                        <Undertittel>Det kan hende du må sende inn dokumentasjon</Undertittel>
+                        <Tekstområde
+                            className={'dokumentasjon__tekstområde'}
+                        >{`Du får beskjed underveis i søknaden hvis du må dokumentere noen av opplysningene dine. Noen ganger kan vi også trenge mer informasjon. Da gir vi deg beskjed om dette.`}</Tekstområde>
+                        <Lenke href={`https://www.nav.no`}>
+                            Oversikt over hva som krever dokumentasjon
+                        </Lenke>
+                    </div>
+
+                    <div className={'informasjon'}>
```

Comment:
Jeg ser det er ganske mange like komponenter. Kunne dere skrevet en komponent som kunne blitt gjenbrukt for styling og tekst/children?